src/wallet/wallet.h

```diff
@@ -1141,15 +1141,15 @@ class CReserveKey final : public CReserveScript
 
 
 /** 
- * Account information.
- * Stored in wallet with key "acc"+string account name.
+ * Label information.
```

Comment:
This structure is account-specific. It's going away when account support is dropped, as there is no state per label, it's just an identifier. So I suggest keeping it at the current name.